src/core/bot/transit.py

```
@@ -0,0 +1,208 @@
+        return None
+
+    except Exception as e:
+        message.report_exception('Core.Transit', traceback.format_exc(), e)
+        return None
+
+
+def clear_message_queue():
+    global _terminate_signal
+    with _terminate_lock:
+        _terminate_signal = True
+    for module, module_query_queue in _query_queue.items():
```

Ruff (0.11.9)

Loop control variable `module` not used within loop body

Rename unused `module` to `_module`

(B007)